pkg/openapi/openapi_generated.go

```diff
@@ -422,7 +422,7 @@ func GetOpenAPIDefinitions(ref openapi.ReferenceCallback) map[string]openapi.Ope
 		"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1.ClusterServicePlanList": {
 			Schema: spec.Schema{
 				SchemaProps: spec.SchemaProps{
-					Description: "ServicePlanList is a list of ServicePlans.",
```

Comment:
This is odd, did something not get generated properly the first go around?

Reply:
Yeah, we need a verify script.

Reply:
Looks like the description it's being generated from needs to change ServicePlans to ClusterServicePlans.